codecept.conf.js

```diff
@@ -0,0 +1,34 @@
+exports.config = {
+  tests: "tests/**/*.js",
+  timeout: 10000,
+  output: "./output",
+  helpers: {
+    Puppeteer: {
+      url: "https://www-ccd.nonprod.platform.hmcts.net/list/case?jurisdiction=DIVORCE&case-type=DIVORCE&case-state=Submitted",
```

Comment:
Please move urls and credentials to separate file

Reply:
ok

Reply:
Tests now depend on a TEST_ENV variable.